pleskdistup/actions/distupgrade.py

```diff
@@ -145,6 +147,97 @@ def estimate_revert_time(self) -> int:
         return 20
 
 
+class SetupAptReposRegexp(action.ActiveAction):
+    from_regexp: str
+    to_regexp: str
+    sources_list_path: str
+    sources_list_d_path: str
+    backup_suffix: str
+    _name: str
+
+    def __init__(
+        self,
+        from_regexp: str,
+        to_regexp: str,
+        sources_list_path: str = "/etc/apt/sources.list",
+        sources_list_d_path: str = "/etc/apt/sources.list.d/",
+        name: str = "set up APT repositories to change from {self.from_regexp!r} to {self.to_regexp!r}",
+    ):
+        self.from_regexp = from_regexp
+        self.to_regexp = to_regexp
+        self.sources_list_path = sources_list_path
+        self.sources_list_d_path = sources_list_d_path
+
+        self._name = name
+        self.backup_suffix = "_" + str(uuid.uuid4())
```

Comment:
What is the purpose of configuring a backup suffix?

Reply:
Not to overwrite it with a well known suffix (like `.bak`)